gengokit/httptransport/httptransport.go

```diff
@@ -285,6 +289,9 @@ if err != nil {
 // createDecodeTypeConversion creates a go string that converts a 64 bit type to a 32 bit type
 // as strconv.ParseInt, ParseUInt, and ParseFloat always return the 64 bit type
 func createDecodeTypeConversion(f Field) string {
+	if f.Repeated {
```

Comment:
Possibly a very short comment about why this early exit. I can kind of understand why, but above it talks about string convert for 64 bit types and I am confused how a repeated type got in here.